pkg/features/system/collector.go

```diff
@@ -369,3 +391,35 @@ func (c *systemCollector) collectSatelites(client *rpc.Client, ch chan<- prometh
 		ch <- prometheus.MustNewConstMetric(hardwareInfoDesc, prometheus.GaugeValue, float64(1), l...)
 	}
 }
+
+func (c *systemCollector) collectLicense(client *rpc.Client, ch chan<- prometheus.Metric, labelValues []string) {
+	r := &licenseInformation{}
+	err := client.RunCommandAndParse("show system license usage", r)
+
+	if err == nil {
```

Comment:
consider returning early here on ```err != nil``` to reduce the number of levels used for better readability